### PR TITLE
Added arguments to build custom presets for converting videos

### PR DESCRIPTION
**Pull request by max619**

### CODE REVIEW

File: FFMpegSharp.Tests/VideoTest.cs [hunk 1]
```diff
@@ -54,23 +89,57 @@ public void Video_ToMP4()
...
         {
             Assert.IsTrue(Convert(VideoType.Ts));
         }
 
+        [TestMethod]
+        public void Video_ToTS_Args()
+        {
+            var container = new ArgumentsContainer();
+            container.Add(new VideoCodecArgument(VideoCodec.MpegTs));
+            Assert.IsTrue(Convert(VideoType.Ts));
```
Comment (vladjerca) [inline FFMpegSharp.Tests/VideoTest.cs]: The `container` doesn't seem to be used here.

File: FFMpegSharp.Tests/VideoTest.cs [hunk 2]
```diff
@@ -40,6 +41,40 @@ public bool Convert(VideoType type, bool multithreaded = false, VideoSize size =
...
+                var input = VideoInfo.FromFileInfo(Input);
+
+                container.Add(new InputArgument(input));
+                container.Add(new OutputArgument(output));
+
+                Encoder.Convert(container);
+
+                var outputVideo = new VideoInfo(output.FullName);
+
+                var b = File.Exists(output.FullName) &&
```
Comment (vladjerca) [inline FFMpegSharp.Tests/VideoTest.cs]: I think we could return the result directly, `b` doesn't seem to be used.

File: FFMpegSharp.Tests/VideoTest.cs [hunk 3]
```diff
@@ -25,7 +26,7 @@ public bool Convert(VideoType type, bool multithreaded = false, VideoSize size =
 
                 var outputVideo = new VideoInfo(output.FullName);
 
-                return File.Exists(output.FullName) &&
+                var b = File.Exists(output.FullName) &&
```
Comment (vladjerca) [inline FFMpegSharp.Tests/VideoTest.cs]: Same here, we could directly return the result.

File: FFMpegSharp.Tests/VideoTest.cs [hunk 4]
```diff
@@ -54,23 +89,57 @@ public void Video_ToMP4()
...
+            container.Add(new VideoCodecArgument(VideoCodec.MpegTs));
+            Assert.IsTrue(Convert(VideoType.Ts));
+        }
+
 
         [TestMethod]
         public void Video_ToOGV_Resize()
         {
-            Assert.IsTrue(Convert(VideoType.Ogv, true, VideoSize.Ed));
+            Assert.IsFalse(Convert(VideoType.Ogv, true, VideoSize.Ed));
```
Comment (vladjerca) [inline FFMpegSharp.Tests/VideoTest.cs]: I think these tests should assert if the conversion is successful. `IsTrue` should pass for all conversion options, if the arguments are required in order for the conversion to be successful, the API will break.

**Comment by vladjerca** [comment]
@max619 If you want I could merge your PR and take care of the other issues myself. Just let me know :+1:

**Comment by vladjerca** [comment]
@max619 Not a problem, Take your time 💯

**Comment by max619** [comment]
@vladjerca , well, looks like it is done, if you think that we need to change something in this pull request, let me know :)

**Pull request merged**